src/cdk/scrolling/virtual-for-of.ts

```diff
@@ -166,14 +167,15 @@ export class CdkVirtualForOf<T> implements CollectionViewer, DoCheck, OnDestroy
       /** The set of available differs. */
       private _differs: IterableDiffers,
       /** The virtual scrolling viewport that these items are being rendered in. */
-      @SkipSelf() private _viewport: CdkVirtualScrollViewport) {
+      @SkipSelf() private _viewport: CdkVirtualScrollViewport,
+      ngZone: NgZone) {
```

Comment:
I left this before, but I think it got lost between commits: shouldn't we make this an optional parameter with a breaking change for the next major?

Reply:
virtual scroll isn't released yet, so it's not necessary